js/data.js

```diff
@@ -135,13 +135,13 @@ var data = {
 		kurz: "BD Analytics",
 		lehrform: ["Vorlesung", "Übung"],
 		modul: ["Vertiefungsgebiete"],
-		semester: [ws15_16],
+		semester: [ws15_16, ss16],
```

Comment:
This is a different course, it's a lab and not a lecture. We have to make this a new course, as otherwise you cannot choose both.